src/main/java/com/exmertec/dummie/DummyBuilderFactory.java

```diff
@@ -0,0 +1,29 @@
+package com.exmertec.dummie;
+
+
+import com.exmertec.dummie.cache.DummyCache;
+import com.exmertec.dummie.cache.impl.DefaultCache;
+import com.exmertec.dummie.cache.impl.LevelCache;
+
+public class DummyBuilderFactory {
```

Comment:
i think it depends on which to config.The cache that DummyBuilder used is important and unchanged so could be provided when inited